src/renderers/native/ReactNativeStack.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactNativeStack
+ * @flow
+ */
+'use strict';
+
+// Require ReactNativeDefaultInjection first for its side effects of setting up
```

Comment:
This was copied as-is from the pre-existing [`ReactNative.js`](https://github.com/facebook/react/blob/ba8f24ba9937c6cec70e4cb00aa2ded02f7d5cdc/src/renderers/native/ReactNative.js#L14-L15). The only modifications to this file was the `@providesModule` name and the 6-line `findNodeHandle.injection` calls.

I'll delete the comment though.